pkg/authorization/authz_unix_test.go

```diff
@@ -257,20 +249,14 @@ func (t *authZPluginTestServer) stop() {
 // auth is a used to record/replay the authentication api messages
 func (t *authZPluginTestServer) auth(w http.ResponseWriter, r *http.Request) {
 	t.recordedRequest = Request{}
-	defer r.Body.Close()
-	body, err := ioutil.ReadAll(r.Body)
+	body, _ := ioutil.ReadAll(r.Body)
```

Comment:
True but you could return a error (in the response) and thus make the test fail if an error occured (and was not wanted) :angel: